source/rmarkdown/metadata_bodemstalen_fysicochemisch.Rmd

```diff
@@ -38,49 +49,118 @@ file_name <- "MBAG_stratfile_v2_cleaned_12.csv"
 
 df_numeric_clean <- df_numeric[, apply(df_numeric, 2, var, na.rm = TRUE) != 0]
 
+# de Lambert coordinaten laten we ook buiten beschouwing
+df_numeric_clean <- df_numeric_clean %>%
+  select(-c_LB72X, -c_LB72Y)
 ```
 
-Pairwise correlations
 
-```{r pairwise-correlations}
-cor_matrix <- cor(df_numeric_clean, use = "pairwise.complete.obs")
-
-png("correlation_plot.png", width = 1200, height = 1000)
-corrplot(cor_matrix, method = "color", type = "upper", order = "hclust", 
-         tl.col = "black", tl.srt = 45, tl.cex = 0.7)
-dev.off()
+```{r summary-stats}
+summary_stats <- df_numeric_clean %>%
+  tidyr::pivot_longer(cols = everything()) %>%
+  summarise(
+    mean = mean(value, na.rm = TRUE),
+    sd = sd(value, na.rm = TRUE),
+    min = min(value, na.rm = TRUE),
+    max = max(value, na.rm = TRUE),
+    skewness = moments::skewness(value, na.rm = TRUE),
+    kurtosis = moments::kurtosis(value, na.rm = TRUE),
+    .by = c(name)
+  )
+kable(summary_stats, digits = 2)
 ```
 
+Histogrammen en qq-plots kunnen helpen om in te schatten hoe de data verdeeld zijn en of een transformatie van de data nuttig kan zijn.
+Normaliteit is echter geen vereiste om verklarende variabelen te gebruiken in een regressiemodel.
+Enkel de residuen van een (veralgemeend lineair) regressiemodel moeten normaal verdeeld zijn.
 
-```{r dispay-corrplot}
-# Display correlation plot
-plot(1:10, type = "n", xlab = "", ylab = "", main = "Correlation Plot")
-img <- png::readPNG("correlation_plot.png")
-grid::grid.raster(img)
+
+```{r histogrammen}
+df_numeric_clean %>%
+  tidyr::pivot_longer(cols = everything()) %>%
+  ggplot() +
+  geom_histogram(
+    aes(x = value)
+  ) +
+  facet_wrap(~ name, scales = "free")
 ```
 
-Multicollinearity test
+```{r qqplots}
+df_numeric_clean %>%
+  tidyr::pivot_longer(cols = everything()) %>%
+  ggplot(aes(sample = value)) +
+  stat_qq_line() +
+  stat_qq() +
+  facet_wrap(~ name, scales = "free")
+```
 
-```{r multicollinearity}
-constant_response <- rep(1, nrow(df_numeric_clean))
-vif_model <- lm(constant_response ~ ., data = df_numeric_clean)
-vif_results <- vif(vif_model)
+De textuurvariabelen zijn een voorbeeld van compositionele data (net zoals eDNA reads) en kunnen we beter voorstellen in een textuurdriehoek:
+
+```{r ternaryplot}
+ggtern::ggtern(
+  data = df,
+  ggtern::aes(Textuur_Zandfractie, Textuur_kleifractie, Textuur_leemfractie)) + 
+   ggtern::geom_mask() +
+   geom_point(
+     ggtern::aes(
+       colour = Landgebruik_MBAG,
+       shape = Landgebruik_MBAG)) + 
+   ggtern::theme_ggtern() +
+   ggtern::theme_showarrows() +
+   ggtern::theme_clockwise()
 ```
 
-```{r display-vif-results}
-print(vif_results)
+## Multicollineariteit
+
+We checken multicollineariteit tussen covariaten.
+Significante paarsgewijze correlaties met absolute waarden groter dan 0.7 kunnen op problemen wijzen.
+Variance inflation factoren groter dan 5 zijn mogelijk problematisch en groter dan 10 zijn zeker problematisch.
+
```

Comment:
Moeten we eigenlijk ook correlatie testen tussen de niet-numerieke variabelen (zoals landgebruik en diepte) en de numerieke variabelen?

Reply:
Ja. Ik zou gewoon grafieken maken en visueel beoordelen. Bijvoorbeeld met [ggforce::geom_sina()](https://ggforce.data-imaginist.com/reference/geom_sina.html) + geom_violin() (factor in x as, continue in y as)